test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/BusinessEntityControllerSpec.scala

```diff
@@ -114,47 +118,77 @@ class BusinessEntityControllerSpec
 
       "display the page" when {
 
-        def test(
-          input: Option[String],
-          businessEntityJourney: BusinessEntityJourney,
-          undertaking: Undertaking
-        ): Unit = {
-          inSequence {
-            mockAuthWithEnrolmentAndValidEmail()
-            mockRetrieveUndertaking(eori1)(undertaking.some.toFuture)
-            mockGetOrCreate[BusinessEntityJourney](eori1)(Right(businessEntityJourney))
-          }
+        "user has not already answered the question - no added business entities added" in new AddBusinessPageSetup(
+          theUndertaking = undertaking.copy(undertakingBusinessEntity = List(businessEntity1)),
+          theBusinessEntityJourney = businessEntityJourney.copy(addBusiness = AddBusinessFormPage())
+        ) {
 
-          checkPageIsDisplayed(
-            performAction,
-            if (undertaking.undertakingBusinessEntity.size > 1)
-              messageFromMessageKey("addBusiness.businesses-added.legend")
-            else messageFromMessageKey("addBusiness.legend"),
-            { doc =>
-              val selectedOptions = doc.select(".govuk-radios__input[checked]")
+          val result = performAction
+          status(result) shouldBe OK
+          val document = Jsoup.parse(contentAsString(result))
 
-              input match {
-                case Some(value) => selectedOptions.attr("value") shouldBe value
-                case None => selectedOptions.isEmpty shouldBe true
-              }
-
-              val button = doc.select("form")
-              button.attr("action") shouldBe routes.BusinessEntityController.postAddBusinessEntity.url
-            }
-          )
+          verifyAddBusinessPageCommonElements(document)
+          document
+            .getElementById("no-other-businesses")
+            .text() shouldBe "Other than the undertaking administrator, there are currently no other businesses in your undertaking."
+          document.getElementById("addBusiness").hasAttr("checked") shouldBe false
```

Comment:
Possibly we should look at using a page object in the future as they are resistant to style changes etc.

we can talk about that later :)